index.js

```diff
@@ -44,7 +46,15 @@ module.exports = {
 
   treeForAddonTestSupport: function() {
     var MergeTrees = require('broccoli-merge-trees');
-    return new MergeTrees(this._getDependencyTrees());
+    var tree = new MergeTrees(this._getDependencyTrees());
+
+    if (this._shouldPreprocessAddonTestSupport) {
+      return this.preprocessJs(tree, '/', this.name, {
```

Comment:
Turns out (after reviewing with @trentmwillis separately) that ember-cli-babel does not actually use any param other than `tree`.  We should remove the others here to prevent my own future confusion 😝